frontend/src/components/TextInputComponent.vue

```diff
@@ -6,9 +6,10 @@
       :maxLength="maxLength"
       :placeholder="placeHolder"
       :disabled="disabled"
-      v-model="inputText"
+      :value="props.modelValue"
```

Comment:
I dont think this is needed anymore, from my testing

Reply:
hm, thats right i guess the hot reload from vite my not work for this right